pgpy/types.py

```diff
@@ -59,7 +59,7 @@ class Armorable(six.with_metaclass(abc.ABCMeta)):
     __armor_regex = re.compile(r"""# This capture group is optional because it will only be present in signed cleartext messages
                          (^-{5}BEGIN\ PGP\ SIGNED\ MESSAGE-{5}(?:\r?\n)
                           (Hash:\ (?P<hashes>[A-Za-z0-9\-,]+)(?:\r?\n){2})?
-                          (?P<cleartext>(.*\r?\n)+?)(?:\r?\n)?
+                          (?P<cleartext>(\r?\n|.)+?)(?:\r?\n)?
```

Comment:
Sure I can use that, although it's a bit more complex. The armor regex shouldn't change once it's "right" so that should not be a problem. Just pushed with this regex and a one additional cleartext test.